Gemfile

```diff
@@ -44,7 +44,8 @@ gem 'laa-criminal-applications-datastore-api-client',
     require: 'datastore_api'
 
 gem 'laa-criminal-legal-aid-schemas',
-    github: 'ministryofjustice/laa-criminal-legal-aid-schemas', tag: 'v1.0.0'
+    github: 'ministryofjustice/laa-criminal-legal-aid-schemas', ref: 'c16fa1a'
```

Comment:
Last commit ref in the gem is `09449a6` but perhaps best to cut a new version once the schemas PR is merged and use it here 👍